packages/web-forms/src/components/controls/RankControl.vue

```diff
@@ -101,58 +107,70 @@ const swapItems = (index: number, newPosition: number) => {
 <template>
 	<ControlText :question="question" />
 
-	<VueDraggable
```

Comment:
Anything inside the `VueDraggable` element is the same as before; I just moved it inside a `rank-control-container`. The structure is:

|- rank-control-container
   |- rank overlay 
   |- VueDraggable

The rank overlay isn't inside `VueDraggable` because it's not good that it inherits "draggable" styles or events when it's not a draggable element.